settings.talon

```diff
@@ -24,6 +24,9 @@ settings():
     # Set the scroll amount for continuous scroll/gaze scroll
     user.mouse_continuous_scroll_amount = 80
 
+    # Set the maximum acceleration factor when scrolling continuously. 1=constant speed/no acceleration.
+    user.mouse_continuous_scroll_acceleration = 1
```

Comment:
Could we document the type of acceleration (ie linear)?

Reply:
done